packages/toolkit/src/view/hub/hub-view/NewsLetterCard.tsx

```diff
@@ -2,7 +2,7 @@
 
 import { Icons, Skeleton, buttonVariants } from "@instill-ai/design-system";
 import { ImageWithFallback } from "../../../components/ImageWithFallback";
-import { useState } from "react";
+import { useEffect, useState } from "react";
```

Comment:
Just a unification of code style, 

we use `import * as React from "react"` among all the codebase, please change this in hub related components